src/panels/history/ha-panel-history.ts

```diff
@@ -211,12 +227,35 @@ class HaPanelHistory extends LitElement {
           position: relative;
         }
 
+        ha-date-range-picker {
+          margin-right: 16px;
+          max-width: 100%;
+        }
+
+        :host([narrow]) ha-date-range-picker {
+          margin-right: 0;
+        }
+
         ha-circular-progress {
           position: absolute;
           left: 50%;
           top: 50%;
           transform: translate(-50%, -50%);
         }
+
+        ha-entity-picker {
+          display: inline-block;
+          flex-grow: 1;
+          max-width: 400px;
+          --paper-input-suffix: {
+            height: 24px;
+          }
```

Comment:
I don't think this works everywhere, why is it needed?

Reply:
Honest answer: 1:1 copy from the logbook panel 😄. I assumed if it is in there, we need it here as well. I can test without for both panels.